cocos/ui/UIEditBox/UIEditBoxImpl-common.cpp

```diff
@@ -95,8 +99,39 @@ void EditBoxImplCommon::initInactiveLabels(const Size& size)
 
 void EditBoxImplCommon::placeInactiveLabels()
 {
-    _label->setPosition(CC_EDIT_BOX_PADDING, _contentSize.height / 2.0f);
-    _labelPlaceHolder->setPosition(CC_EDIT_BOX_PADDING, _contentSize.height / 2.0f);
+    switch (_textHorizontalAlignMent) {
+        case cocos2d::TextHAlignment::LEFT :
```

Comment:
@anjandaffo  
In the method `placeInactiveLabels`, we should also call `label->setTextAlignment` related API.